src/Types.jl

```diff
@@ -336,6 +336,7 @@ Attributes:
 immutable SkyPatch
     center::Vector{Float64}
     radius::Float64
```

Comment:
If you've done that then go ahead and delete it.

-- Ryan.  Sent from a phone.  Nevertheless, all typos are deliberate.
On Mar 18, 2016 12:55 PM, "Ryan Giordano" <rgiordan@gmail.com> wrote:

> I'm in a meeting so I can't look at the code.  But before deleting it
> could you make sure something similar is factored out of infer() so that I
> can use it interactively?
>
> -- Ryan.  Sent from a phone.  Nevertheless, all typos are deliberate.
> On Mar 18, 2016 12:09 PM, "Kyle Barbary" <notifications@github.com> wrote:
>
>> In src/Types.jl
>> <https://github.com/jeff-regier/Celeste.jl/pull/164#discussion_r56707521>
>> :
>>
>> > @@ -336,6 +336,7 @@ Attributes:
>> >  immutable SkyPatch
>> >      center::Vector{Float64}
>> >      radius::Float64
>>
>> Yes, will do. Similarly, initialize_objid is no longer used. Can we
>> delete it?
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/jeff-regier/Celeste.jl/pull/164/files/5b88c0642fda9e89fbf5a1156dd2bfbea09d6767#r56707521>
>>
>